code-challenges.js

```diff
@@ -31,24 +36,57 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+const boilingTemp = (temp) => {
+    if (temp > 212 ) {
+        return `${temp} is above boiling point.`
+    } else if (temp < 212) {
+        return `${temp} is below boiling point.`
+    } else {
+        return `${temp} is at boiling point.`
+    }
+}
+console.log(boilingTemp(temperature1)) // output: "42 is below boiling point."
```

Comment:
One thing to consider would be what would happen if we passed a string as the argument. What would the return be?